### ISSUE TITLE
Nested files don't get passed '~' option

### ISSUE THREAD

**Issue by richardbutler** [jtangelder/sass-loader#108]
If I have a library in `node_modules`, which is imported by a partial, then imported again, I get `ERROR in  ... File to import not found or unreadable`.

Given `node_modules/somelib/colours.scss` exists, this doesn't work:

>sass/core/_colours.scss:
```scss
@import '~somelib/colours.scss';
```

>src/components/Thing.scss:
```scss
@import 'core/colours';
```

>src/components/Thing.jsx:
```js
require('./Thing.scss');
```

However, this works:

>src/components/Thing.scss:
```scss
@import '~somelib/colours.scss';
```

>src/components/Thing.jsx:
```js
require('./Thing.scss');
```

Note I'm running on Windows 7.

**Comment by richardbutler** [comment]
Possibly related to #102 and #107.

**Comment by jhnns** [comment]
Thx for reporting this. I'll have a look as soon as I found some time. Unfortunately I'm loaded with work... :unamused:.

In the meantime, you can also take a look for yourself. The code is not complicated, but it requires some understanding about how to write loaders for webpack.

**Comment by jhnns** [comment]
Is this still an issue with the recent version of sass-loader?

**Comment by jhnns** [comment]
Should be fixed with 2.0.0
